.github/workflows/publish.yml

```diff
@@ -0,0 +1,57 @@
+---
+name: Release
+
+on:
+  release:
+    types:
+      - published
+
+concurrency:
+  group: release-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  hackage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Haskell
+        id: setup-haskell-cabal
+        uses: haskell/actions/setup@v2
+        with:
+          # I faced the following issue with GHC-9.2 and and I managed to solved
+          # it by going back to GHC-8.10.7 just to generate the docs:
+          # - https://github.com/haskell/cabal/issues/8326
+          ghc-version: "8.10.7"
+          cabal-version: "3.6"
+
+      - name: Generate sdist
+        run: cabal sdist
+
+      - name: Upload candidate package to Hackage
+        env:
+          HACKAGE_API_KEY: ${{ secrets.HACKAGE_API_KEY }}
+        run: |
+          SDIST_PATH=$(ls dist-newstyle/sdist/cassava-megaparsec-*.tar.gz)
+          curl -X POST \
+           -H 'Accept: text/plain' \
+           -H "Authorization: X-ApiKey \"$HACKAGE_API_KEY\"" \
+           --form "package=@${SDIST_PATH}" \
+           "https://hackage.haskell.org/packages/candidates"
+      - name: Generate documentation
+        run: cabal haddock --haddock-for-hackage --enable-documentation
+
+      - name: Upload Docs for candidate to Hackage
+        env:
+          HACKAGE_API_KEY: ${{ secrets.HACKAGE_API_KEY }}
+        run: |
+          TAG_VERSION=${{ github.ref_name }}
+          VERSION=${TAG_VERSION:1}
```

Comment:
Hey @cptrodolfox I did this in dotenv because the tag versions start with a `v` and I needed only the PVP. I see cassava-megaparsec [tags the versions](https://github.com/stackbuilders/cassava-megaparsec/tags) only with the PVP. Therefore, you can remove this line.

Reply:
@CristhianMotoche Thanks for the heads up. I have removed the line.